Patches/Dragon's Descent/ThingDefs_Races/Patch_Races_Animal_Rare_Dragon.xml

```diff
@@ -0,0 +1,706 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- Making the common dragons CE compatible -->
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+			<!-- ==== Check for the mod. ==== -->
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Dragon's Descent</modName>
+			</li>
+
+			<!-- Rare dragon base. -->
+			<!-- Giving it a shape. -->
+			<li Class="PatchOperationAddModExtension">
+				<xpath>Defs/ThingDef[@Name="RDragonRaceBase"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Quadruped</bodyShape>
```

Comment:
As above, I'd investigate using `BirdLike` which is used for Thrumbos.

Reply:
From examining the texture of the dragons and thrumbos, I'll switch it over `BirdLike`.